Fri Aug 02 22:05:17 MDT 2013 Thread[main,5,main] Cleanup action starting
java.sql.SQLException: Database 'test' not found.
	at org.apache.derby.impl.jdbc.SQLExceptionFactory40.getSQLException(Unknown Source)
	at org.apache.derby.impl.jdbc.Util.newEmbedSQLException(Unknown Source)
	at org.apache.derby.impl.jdbc.Util.newEmbedSQLException(Unknown Source)
	at org.apache.derby.impl.jdbc.Util.generateCsSQLException(Unknown Source)
	at org.apache.derby.impl.jdbc.EmbedConnection.newSQLException(Unknown Source)
	at org.apache.derby.impl.jdbc.EmbedConnection.handleDBNotFound(Unknown Source)
	at org.apache.derby.impl.jdbc.EmbedConnection.<init>(Unknown Source)
	at org.apache.derby.impl.jdbc.EmbedConnection40.<init>(Unknown Source)
	at org.apache.derby.jdbc.Driver40.getNewEmbedConnection(Unknown Source)
	at org.apache.derby.jdbc.InternalDriver.connect(Unknown Source)
	at org.apache.derby.jdbc.Driver20.connect(Unknown Source)
	at org.apache.derby.jdbc.EmbeddedDriver.connect(Unknown Source)
	at org.hibernate.service.jdbc.connections.internal.DriverManagerConnectionProviderImpl.getConnection(DriverManagerConnectionProviderImpl.java:204)
	at org.hibernate.engine.jdbc.internal.JdbcServicesImpl$ConnectionProviderJdbcConnectionAccess.obtainConnection(JdbcServicesImpl.java:242)
	at org.hibernate.engine.jdbc.internal.JdbcServicesImpl.configure(JdbcServicesImpl.java:117)
	at org.hibernate.service.internal.StandardServiceRegistryImpl.configureService(StandardServiceRegistryImpl.java:75)
	at org.hibernate.service.internal.AbstractServiceRegistryImpl.initializeService(AbstractServiceRegistryImpl.java:159)
	at org.hibernate.service.internal.AbstractServiceRegistryImpl.getService(AbstractServiceRegistryImpl.java:131)
	at org.hibernate.cfg.Configuration.buildTypeRegistrations(Configuration.java:1797)
	at org.hibernate.cfg.Configuration.buildSessionFactory(Configuration.java:1755)
	at org.hibernate.ejb.EntityManagerFactoryImpl.<init>(EntityManagerFactoryImpl.java:96)
	at org.hibernate.ejb.Ejb3Configuration.buildEntityManagerFactory(Ejb3Configuration.java:914)
	at org.hibernate.ejb.Ejb3Configuration.buildEntityManagerFactory(Ejb3Configuration.java:899)
	at org.hibernate.ejb.HibernatePersistence.createEntityManagerFactory(HibernatePersistence.java:59)
	at javax.persistence.Persistence.createEntityManagerFactory(Persistence.java:63)
	at com.google.inject.persist.jpa.JpaPersistService.start(JpaPersistService.java:97)
	at com.jshnd.assassin.persistence.jpa.JpaAssassinStoreIntegrationTest$$anonfun$1.apply$mcV$sp(JpaAssassinStoreIntegrationTest.scala:14)
	at com.jshnd.assassin.persistence.jpa.JpaAssassinStoreIntegrationTest$$anonfun$1.apply(JpaAssassinStoreIntegrationTest.scala:12)
	at com.jshnd.assassin.persistence.jpa.JpaAssassinStoreIntegrationTest$$anonfun$1.apply(JpaAssassinStoreIntegrationTest.scala:12)
	at org.scalatest.FunSpec$$anon$1.apply(FunSpec.scala:1600)
	at org.scalatest.Suite$class.withFixture(Suite.scala:1974)
	at com.jshnd.assassin.persistence.jpa.JpaAssassinStoreIntegrationTest.withFixture(JpaAssassinStoreIntegrationTest.scala:10)
	at org.scalatest.FunSpec$class.invokeWithFixture$1(FunSpec.scala:1597)
	at org.scalatest.FunSpec$$anonfun$runTest$1.apply(FunSpec.scala:1606)
	at org.scalatest.FunSpec$$anonfun$runTest$1.apply(FunSpec.scala:1606)
	at org.scalatest.SuperEngine.runTestImpl(Engine.scala:198)
	at org.scalatest.FunSpec$class.runTest(FunSpec.scala:1606)
	at com.jshnd.assassin.persistence.jpa.JpaAssassinStoreIntegrationTest.runTest(JpaAssassinStoreIntegrationTest.scala:10)
	at org.scalatest.FunSpec$$anonfun$runTests$1.apply(FunSpec.scala:1639)
	at org.scalatest.FunSpec$$anonfun$runTests$1.apply(FunSpec.scala:1639)
	at org.scalatest.SuperEngine$$anonfun$org$scalatest$SuperEngine$$runTestsInBranch$1.apply(Engine.scala:260)
	at org.scalatest.SuperEngine$$anonfun$org$scalatest$SuperEngine$$runTestsInBranch$1.apply(Engine.scala:249)
	at scala.collection.immutable.List.foreach(List.scala:318)
	at org.scalatest.SuperEngine.org$scalatest$SuperEngine$$runTestsInBranch(Engine.scala:249)
	at org.scalatest.SuperEngine.runTestsImpl(Engine.scala:326)
	at org.scalatest.FunSpec$class.runTests(FunSpec.scala:1639)
	at com.jshnd.assassin.persistence.jpa.JpaAssassinStoreIntegrationTest.runTests(JpaAssassinStoreIntegrationTest.scala:10)
	at org.scalatest.Suite$class.run(Suite.scala:2303)
	at com.jshnd.assassin.persistence.jpa.JpaAssassinStoreIntegrationTest.org$scalatest$FunSpec$$super$run(JpaAssassinStoreIntegrationTest.scala:10)
	at org.scalatest.FunSpec$$anonfun$run$1.apply(FunSpec.scala:1645)
	at org.scalatest.FunSpec$$anonfun$run$1.apply(FunSpec.scala:1645)
	at org.scalatest.SuperEngine.runImpl(Engine.scala:362)
	at org.scalatest.FunSpec$class.run(FunSpec.scala:1645)
	at com.jshnd.assassin.persistence.jpa.JpaAssassinStoreIntegrationTest.run(JpaAssassinStoreIntegrationTest.scala:10)
	at org.scalatest.tools.SuiteRunner.run(SuiteRunner.scala:60)
	at org.scalatest.tools.Runner$$anonfun$doRunRunRunDaDoRunRun$3.apply(Runner.scala:1604)
	at org.scalatest.tools.Runner$$anonfun$doRunRunRunDaDoRunRun$3.apply(Runner.scala:1601)
	at scala.collection.immutable.List.foreach(List.scala:318)
	at org.scalatest.tools.Runner$.doRunRunRunDaDoRunRun(Runner.scala:1601)
	at org.scalatest.tools.Runner$$anonfun$runOptionallyWithPassFailReporter$2.apply(Runner.scala:705)
	at org.scalatest.tools.Runner$$anonfun$runOptionallyWithPassFailReporter$2.apply(Runner.scala:704)
	at org.scalatest.tools.Runner$.withClassLoaderAndDispatchReporter(Runner.scala:1645)
	at org.scalatest.tools.Runner$.runOptionallyWithPassFailReporter(Runner.scala:703)
	at org.scalatest.tools.Runner$.run(Runner.scala:592)
	at org.scalatest.tools.Runner.run(Runner.scala)
	at org.jetbrains.plugins.scala.testingSupport.scalaTest.ScalaTestRunner.runScalaTest1(ScalaTestRunner.java:213)
	at org.jetbrains.plugins.scala.testingSupport.scalaTest.ScalaTestRunner.main(ScalaTestRunner.java:37)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:57)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:606)
	at com.intellij.rt.execution.application.AppMain.main(AppMain.java:120)
Caused by: java.sql.SQLException: Database 'test' not found.
	at org.apache.derby.impl.jdbc.SQLExceptionFactory.getSQLException(Unknown Source)
	at org.apache.derby.impl.jdbc.SQLExceptionFactory40.wrapArgsForTransportAcrossDRDA(Unknown Source)
	... 72 more
============= begin nested exception, level (1) ===========
java.sql.SQLException: Database 'test' not found.
	at org.apache.derby.impl.jdbc.SQLExceptionFactory.getSQLException(Unknown Source)
	at org.apache.derby.impl.jdbc.SQLExceptionFactory40.wrapArgsForTransportAcrossDRDA(Unknown Source)
	at org.apache.derby.impl.jdbc.SQLExceptionFactory40.getSQLException(Unknown Source)
	at org.apache.derby.impl.jdbc.Util.newEmbedSQLException(Unknown Source)
	at org.apache.derby.impl.jdbc.Util.newEmbedSQLException(Unknown Source)
	at org.apache.derby.impl.jdbc.Util.generateCsSQLException(Unknown Source)
	at org.apache.derby.impl.jdbc.EmbedConnection.newSQLException(Unknown Source)
	at org.apache.derby.impl.jdbc.EmbedConnection.handleDBNotFound(Unknown Source)
	at org.apache.derby.impl.jdbc.EmbedConnection.<init>(Unknown Source)
	at org.apache.derby.impl.jdbc.EmbedConnection40.<init>(Unknown Source)
	at org.apache.derby.jdbc.Driver40.getNewEmbedConnection(Unknown Source)
	at org.apache.derby.jdbc.InternalDriver.connect(Unknown Source)
	at org.apache.derby.jdbc.Driver20.connect(Unknown Source)
	at org.apache.derby.jdbc.EmbeddedDriver.connect(Unknown Source)
	at org.hibernate.service.jdbc.connections.internal.DriverManagerConnectionProviderImpl.getConnection(DriverManagerConnectionProviderImpl.java:204)
	at org.hibernate.engine.jdbc.internal.JdbcServicesImpl$ConnectionProviderJdbcConnectionAccess.obtainConnection(JdbcServicesImpl.java:242)
	at org.hibernate.engine.jdbc.internal.JdbcServicesImpl.configure(JdbcServicesImpl.java:117)
	at org.hibernate.service.internal.StandardServiceRegistryImpl.configureService(StandardServiceRegistryImpl.java:75)
	at org.hibernate.service.internal.AbstractServiceRegistryImpl.initializeService(AbstractServiceRegistryImpl.java:159)
	at org.hibernate.service.internal.AbstractServiceRegistryImpl.getService(AbstractServiceRegistryImpl.java:131)
	at org.hibernate.cfg.Configuration.buildTypeRegistrations(Configuration.java:1797)
	at org.hibernate.cfg.Configuration.buildSessionFactory(Configuration.java:1755)
	at org.hibernate.ejb.EntityManagerFactoryImpl.<init>(EntityManagerFactoryImpl.java:96)
	at org.hibernate.ejb.Ejb3Configuration.buildEntityManagerFactory(Ejb3Configuration.java:914)
	at org.hibernate.ejb.Ejb3Configuration.buildEntityManagerFactory(Ejb3Configuration.java:899)
	at org.hibernate.ejb.HibernatePersistence.createEntityManagerFactory(HibernatePersistence.java:59)
	at javax.persistence.Persistence.createEntityManagerFactory(Persistence.java:63)
	at com.google.inject.persist.jpa.JpaPersistService.start(JpaPersistService.java:97)
	at com.jshnd.assassin.persistence.jpa.JpaAssassinStoreIntegrationTest$$anonfun$1.apply$mcV$sp(JpaAssassinStoreIntegrationTest.scala:14)
	at com.jshnd.assassin.persistence.jpa.JpaAssassinStoreIntegrationTest$$anonfun$1.apply(JpaAssassinStoreIntegrationTest.scala:12)
	at com.jshnd.assassin.persistence.jpa.JpaAssassinStoreIntegrationTest$$anonfun$1.apply(JpaAssassinStoreIntegrationTest.scala:12)
	at org.scalatest.FunSpec$$anon$1.apply(FunSpec.scala:1600)
	at org.scalatest.Suite$class.withFixture(Suite.scala:1974)
	at com.jshnd.assassin.persistence.jpa.JpaAssassinStoreIntegrationTest.withFixture(JpaAssassinStoreIntegrationTest.scala:10)
	at org.scalatest.FunSpec$class.invokeWithFixture$1(FunSpec.scala:1597)
	at org.scalatest.FunSpec$$anonfun$runTest$1.apply(FunSpec.scala:1606)
	at org.scalatest.FunSpec$$anonfun$runTest$1.apply(FunSpec.scala:1606)
	at org.scalatest.SuperEngine.runTestImpl(Engine.scala:198)
	at org.scalatest.FunSpec$class.runTest(FunSpec.scala:1606)
	at com.jshnd.assassin.persistence.jpa.JpaAssassinStoreIntegrationTest.runTest(JpaAssassinStoreIntegrationTest.scala:10)
	at org.scalatest.FunSpec$$anonfun$runTests$1.apply(FunSpec.scala:1639)
	at org.scalatest.FunSpec$$anonfun$runTests$1.apply(FunSpec.scala:1639)
	at org.scalatest.SuperEngine$$anonfun$org$scalatest$SuperEngine$$runTestsInBranch$1.apply(Engine.scala:260)
	at org.scalatest.SuperEngine$$anonfun$org$scalatest$SuperEngine$$runTestsInBranch$1.apply(Engine.scala:249)
	at scala.collection.immutable.List.foreach(List.scala:318)
	at org.scalatest.SuperEngine.org$scalatest$SuperEngine$$runTestsInBranch(Engine.scala:249)
	at org.scalatest.SuperEngine.runTestsImpl(Engine.scala:326)
	at org.scalatest.FunSpec$class.runTests(FunSpec.scala:1639)
	at com.jshnd.assassin.persistence.jpa.JpaAssassinStoreIntegrationTest.runTests(JpaAssassinStoreIntegrationTest.scala:10)
	at org.scalatest.Suite$class.run(Suite.scala:2303)
	at com.jshnd.assassin.persistence.jpa.JpaAssassinStoreIntegrationTest.org$scalatest$FunSpec$$super$run(JpaAssassinStoreIntegrationTest.scala:10)
	at org.scalatest.FunSpec$$anonfun$run$1.apply(FunSpec.scala:1645)
	at org.scalatest.FunSpec$$anonfun$run$1.apply(FunSpec.scala:1645)
	at org.scalatest.SuperEngine.runImpl(Engine.scala:362)
	at org.scalatest.FunSpec$class.run(FunSpec.scala:1645)
	at com.jshnd.assassin.persistence.jpa.JpaAssassinStoreIntegrationTest.run(JpaAssassinStoreIntegrationTest.scala:10)
	at org.scalatest.tools.SuiteRunner.run(SuiteRunner.scala:60)
	at org.scalatest.tools.Runner$$anonfun$doRunRunRunDaDoRunRun$3.apply(Runner.scala:1604)
	at org.scalatest.tools.Runner$$anonfun$doRunRunRunDaDoRunRun$3.apply(Runner.scala:1601)
	at scala.collection.immutable.List.foreach(List.scala:318)
	at org.scalatest.tools.Runner$.doRunRunRunDaDoRunRun(Runner.scala:1601)
	at org.scalatest.tools.Runner$$anonfun$runOptionallyWithPassFailReporter$2.apply(Runner.scala:705)
	at org.scalatest.tools.Runner$$anonfun$runOptionallyWithPassFailReporter$2.apply(Runner.scala:704)
	at org.scalatest.tools.Runner$.withClassLoaderAndDispatchReporter(Runner.scala:1645)
	at org.scalatest.tools.Runner$.runOptionallyWithPassFailReporter(Runner.scala:703)
	at org.scalatest.tools.Runner$.run(Runner.scala:592)
	at org.scalatest.tools.Runner.run(Runner.scala)
	at org.jetbrains.plugins.scala.testingSupport.scalaTest.ScalaTestRunner.runScalaTest1(ScalaTestRunner.java:213)
	at org.jetbrains.plugins.scala.testingSupport.scalaTest.ScalaTestRunner.main(ScalaTestRunner.java:37)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:57)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:606)
	at com.intellij.rt.execution.application.AppMain.main(AppMain.java:120)
============= end nested exception, level (1) ===========
Cleanup action completed
